README.md

```diff
@@ -1 +1,153 @@
-# hello-microvers
\ No newline at end of file
+# hello-microvers
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-Microvers] <a name="about-project"></a>
+
+
+This is the first project for starting with microvers.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- Using linter
+- HTML file
+- CSS file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+ web browser
+### Setup
+
+Clone this repository to your desired folder:
+
+
+  cd hello-microvers
+  git clone git@github.com:nilab-khwaja/hello-microvers.git
+
+### Install
+
+Install this project with:
+
+npm dependencies
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+git init
+get clone http:\\github:nilab-khwaja/hello-microvers.git
```

Comment:
- [ ] I suggest that you use `git` instead of `get`. This will break the execution and therefore the user may not be able to clone the repository as intended 😞 .